src/lib/components/Accordion.react.js

```diff
@@ -9,6 +9,7 @@ import { Accordion as RealComponent } from '../LazyLoader';
  * 	id,
```

Comment:
Suggestion for expanding documentation - more details on how the array corresponds to sections
 @param {{
 * 	id: string,
 * 	accordionHeadings: string[],
 * 	children: React.ReactNode,
 * 	showToggleText: boolean,
 * 	defaultSectionsOpen: boolean[],
 * }} [props={}]
 * @return {React.ReactElement}